Author: Gregor Jasny <gjasny@googlemail.com>
Description: Generate Libs.private fields in pkgconfig files from already defined Makefile variables. 
Forwarded: yes

--- a/lib/libv4l1/Makefile
+++ b/lib/libv4l1/Makefile
@@ -35,7 +35,7 @@
 	@echo 'Version: '$(V4L_UTILS_VERSION) >> libv4l1.pc
 	@echo 'Requires.private: libv4l2' >> libv4l1.pc
 	@echo 'Libs: -L$${libdir} -lv4l1' >> libv4l1.pc
-	@echo 'Libs.private: -lpthread' >> libv4l1.pc
+	@echo 'Libs.private: '$(LIBS_libv4l1) >> libv4l1.pc
 	@echo 'Cflags: -I$${prefix}/include' >> libv4l1.pc
 
 install: all
--- a/lib/libv4l2/Makefile
+++ b/lib/libv4l2/Makefile
@@ -35,7 +35,7 @@
 	@echo 'Version: '$(V4L_UTILS_VERSION) >> libv4l2.pc
 	@echo 'Requires.private: libv4lconvert' >> libv4l2.pc
 	@echo 'Libs: -L$${libdir} -lv4l2' >> libv4l2.pc
-	@echo 'Libs.private: -lpthread' >> libv4l2.pc
+	@echo 'Libs.private: '$(LIBS_libv4l2) >> libv4l2.pc
 	@echo 'Cflags: -I$${prefix}/include' >> libv4l2.pc
 
 install: all
--- a/lib/libv4lconvert/Makefile
+++ b/lib/libv4lconvert/Makefile
@@ -35,7 +35,7 @@
 	@echo 'Description: v4l format conversion library' >> libv4lconvert.pc
 	@echo 'Version: '$(V4L_UTILS_VERSION) >> libv4lconvert.pc
 	@echo 'Libs: -L$${libdir} -lv4lconvert' >> libv4lconvert.pc
-	@echo 'Libs.private: -lrt -lm' >> libv4lconvert.pc
+	@echo 'Libs.private: '$(LIBS_libv4lconvert) >> libv4lconvert.pc
 	@echo 'Cflags: -I$${prefix}/include' >> libv4lconvert.pc
 
 install: all
